src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -584,7 +595,7 @@ void GenerateFieldAccessor(const FieldDescriptor* field, const Options& options,
         "}\n\n",
         "camel_name", UnderscoresToCamelCase(field->name(), true), "number",
         IntToString(field->number()), "deprecation_trigger",
-        deprecation_trigger);
+        deprecation_trigger_with_conditional);
```

Comment:
The field getter in the deprecation trigger conditional here may throw an error since we're not calling `readOneof`

Reply:
fixed in 3d3e299